components/handlers/add_module_handler.py

```diff
@@ -31,5 +31,9 @@ def POST(self):
         module_desc = data.description
         module_mc = data.mc
```

Comment:
Need to wrap line 29 to 32 with a "try: ... except AttributeError:" in case any of the attributes are missing. Refer to module_edit.py to see what I mean.